test/app/main/test-profile-command-handler.js

```diff
@@ -48,6 +48,33 @@ describe('profile-command-handler', () => {
     fs.rmdirSync(tempDir);
   });
 
+  it('Handles save-page commands.', async function () {
+    const reader = {
+      uri: 'http://example.com',
+      title: 'The Éxample',
+      content: '<html><head><h1>Header</h1><p>Content goes here</p></head></html>',
+      textContent: `Header
```

Comment:
Consider testing trimming of the title and the text content, since you take pains to do it.